### PR TITLE
Sensei v1.6 document updates

### PR DESCRIPTION
**Pull request by dlcasto**
Minor updates to the installation process for the Sensei v1.6 documentation including updated screenshots, improved grammar and formatting, and fixed an incorrect "go to section" reference.

### CODE REVIEW

**Comment by AdSchellevis** [comment]
@dlcasto is this ready to merge?

**Comment by dlcasto** [comment]
@AdSchellevis It should be good to go. I only had one minor conflict with the formatting of one header (for some reason) that needed resolved, which I have done.

File: source/vendor/sunnyvalley/sensei.rst [hunk 1]
```diff
@@ -53,15 +53,14 @@ Getting Support
 
 **Support for Free Edition**
 
-If you need help for anything, there is an active discussion about Sensei on OPNsense forum. Feel free to `click here <https://forum.opnsense.org/index.php?topic=9521.new;topicseen#new>`_ and join the conversation.
+If you need help with Sensei, there is an active discussion on the OPNsense forum. Feel free to `click here <https://forum.opnsense.org/index.php?topic=9521.new;topicseen#new>`_ to join the conversation.
```
Comment (muratbalaban43) [inline source/vendor/sunnyvalley/sensei.rst:56]: @dlcasto , I guess we need this new board url: https://forum.opnsense.org/index.php?board=38.0 Thanks to OPNsense team, this is dedicated to the product.

Comment (AdSchellevis) [reply source/vendor/sunnyvalley/sensei.rst:56]: taken care of in https://github.com/opnsense/docs/commit/6ab93d27b29c4b7fca4361ea608e810afbe5419b

**Comment by muratbalaban43** [comment]
@AdSchellevis thank you.

**Pull request merged**